content/en/product_analytics/segmentation/_index.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Segmentation
+kind: documentation
+description: Analyze and understand specific groups or segments of your user base.
+further_reading:
+- link: "/product_analytics/"
+  tag: "Documentation"
+  text: "Product Analytics"
+---
+
+## Overview
+
+Segmentation is a feature in Product Analytics that allows you to analyze and understand specific groups or segments of your user base. By segmenting your users based on various characteristics and behaviors, you can uncover valuable insights, identify trends, and make data-driven decisions to optimize your product and user experience. For example, you can create segment users by purchase amount, active users in a given country, users in a trial, or users in a trial who then converted to a paying organization.
+
+## Build a segment
+
+To create a new segment, navigate to **[Digital Experience Monitoring > Product Analytics > User Segments][1]** from the navigation. You have two sources you can choose from when building a new segment:
+
+- RUM data
+- External data (upload a Reference Table)
+
+{{< img src="product_analytics/segmentation/segmentation-1.png" alt="Create a user segment based on RUM or external data.">}}
+
+### Using RUM data
+
+To create a new segment using RUM data:
+
+1. Choose the user attribute you want to connect your data with. In the below example, it is `usr.id`, but you can use any user attribute available, such as (`usr.email` or `usr.name`).
```

Comment:
Are the parentheses here placed correctly? I'd want to interpret this as users being able to connect on an ORed combination `(usr.email or usr.name)`, in which case I'd put the whole parenthetical in backticks. Otherwise, the parentheses should be removed.